subsys/CMakeLists.txt

```diff
@@ -49,3 +49,4 @@ add_subdirectory(debug)
 add_subdirectory(partition_manager)
 
 add_subdirectory_ifdef(CONFIG_NRF_RPC nrf_rpc)
+add_subdirectory_ifdef(CONFIG_NRF_802154_RADIO_DRIVER ieee802154)
```

Comment:
Should there rather be a Kconfig option to enable FEM configuration? Right now it'd be enabled in any case (this could be a problem on our DKs I guess since they don't have FEM?).

Reply:
In that case `lna_gain_get` just returns 0, as there is no FEM present.